python/src/data.py

```diff
@@ -0,0 +1,46 @@
+import getopt, sys, csv
```

Comment:
Instead of data.py, please change the name of the file to be something more meaningful.